OWNERS_ALIASES

```diff
@@ -0,0 +1,26 @@
+aliases:
+  build-team:
+    - mmorhun
+    - chmeliik
+    - mkosiarc
+    - rcerven
+    - tkdchen
+    - tisutisu
+    - tnevrlka 
+  integration-team:
+    - dirgim
+    - jsztuka
+    - hongweiliu17
+    - Josh-Everett
+    - sonam1412
+    - MartinBasti
```

Comment:
MartinBasti is no longer in the integration team, but in Konflux Build Gold

Reply:
moved him to the under build-team alias